Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1481 @@
+    params = {
+        'path': file_path,
+        'page': page,
+        'pagelen': page_size
+    }
+    excluded_list = []
+    included_list = []
+    if excluded_branches:
+        excluded_list = excluded_branches.split(',')
+    if included_branches:
+        included_list = included_branches.split(',')
+    if not repo:
+        repo = client.repository
+    response = client.commit_list_request(repo, params, excluded_list, included_list)
+    results = check_pagination(client, response, limit)
+    human_readable = []
+    for value in results:
+        d = {'Author': value.get('author').get('raw'),
+             'Commit': value.get('hash'),
+             'Message': value.get('message'),
+             'CreatedAt': value.get('date')}
+        human_readable.append(d)
+
+    headers = ['Author', 'Commit', 'Message', 'CreatedAt']
+    readable_output = tableToMarkdown(
+        name='The list of commits',
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.Commit',
+        outputs=results,
+        raw_response=results
+    )
+
+
+def file_delete_command(client: Client, args: Dict) -> CommandResults:
+    """ Deletes a file in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a success message, in case of a successful action.
+    """
+    repo = args.get('repo')
+    message = args.get('message', '')
+    branch = args.get('branch', '')
+    file_name = args.get('file_name', '')
+    author_name = args.get('author_name', None)
+    author_email = args.get('author_email', None)
+    body = {
+        'message': message,
+        'branch': branch,
+        'files': file_name
+    }
+    if author_name and author_email:
+        body['author'] = f'{author_name} <{author_email}>'
+    if not repo:
+        repo = client.repository
+    response = client.file_delete_request(body, repo)
+    if response.status_code == 201:
+        return CommandResults(readable_output='The file was deleted successfully.')
+    else:
+        raise Exception('The command file-delete failed.')
+
+
+def raw_file_get_command(client: Client, args: Dict) -> List[CommandResults]:
+    """ Returns the content of a given file.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A list containing a CommandResult object and a fileResult object.
+    """
+    repo = args.get('repo')
+    file_path = args.get('file_path', '')
+    branch = args.get('branch', None)
+    params = {
+        'path': file_path
+    }
+    if not repo:
+        repo = client.repository
+    if branch:
+        including_list = [branch]
+    else:
+        including_list = []
+    commit_list = client.commit_list_request(repo=repo, params=params, excluded_list=[], included_list=including_list)
```

Comment:
Is this the way to go?
Need to make 2 calls here?

Reply:
Yes. It is also written in the design.